lib/restrictedsession/restricted.go

```diff
@@ -298,6 +306,27 @@ func (l *auditEventLoop) close() {
 	l.wg.Wait()
 }
 
+// checkLsmBpf checks that eBPF is one of the enable
+// LSM "modules".
+func checkBpfLsm() error {
+	lsmInfo := "/sys/kernel/security/lsm"
+
+	csv, err := ioutil.ReadFile(lsmInfo)
```

Comment:
nit: `ioutils` is deprecated:
```suggestion
	csv, err := os.ReadFile(lsmInfo)
```